docs/data-hub/dataframes.md

```diff
@@ -0,0 +1,79 @@
+# Using Deep Origin DataFrames
+
+This page describes how to use Deep Origin DataFrames, which are the primary object you will use to interact with databases on Deep Origin. This page will cover:
+
+- fetching data from a Deep Origin database
+- modifying data locally
+- writing data back to Deep Origin
+
+
+!!! question "What is a Deep Origin DataFrame?"
```

Comment:
I think we should inline this into the main narrative because this is essential to understanding how this works.

Reply:
> because this is essential to understanding how this works.

i agree. i think of admonitions more as callouts than sidetracks. i'm using it to draw attention to this